<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QPlainTextEdit" name="txt_receber"/>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="bt_limpar">
      <property name="text">
       <string>Limpar</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="bt_enviar">
      <property name="text">
       <string>Enviar</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="txt_enviar"/>
    </item>
    <item row="0" column="0" rowspan="4" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Serial Ports</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_serial"/>
      </item>
      <item>
       <widget class="QComboBox" name="cb_baudrate"/>
      </item>
      <item>
       <widget class="QPushButton" name="bt_open">
        <property name="text">
         <string>Open Serial</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bt_close">
        <property name="text">
         <string>Close Serial</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refresh_bt">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Kick">
        <property name="text">
         <string>KIck</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
